File: Project/Simulator/Organisms/Plants/Milkweed.java
package Simulator.Organisms.Plants;

import org.json.JSONObject;

import Simulator.World;

public class Milkweed extends Plant {

    public Milkweed(World world) {
        super(0, world, Type.MILKWEED);
    }

    public Milkweed(JSONObject json, World world) {
        super(json, world);
    }

    @Override
    public Plant construct() {
        return new Milkweed(world);
    }

    @Override
    public void action() {
        super.action();
        super.action();
        super.action();
    }
}

File: Project/Simulator/Organisms/Plants/Milkweed.py
from .Plant import Plant
from Simulator.Organisms.Organism import Type, Organism


class Milkweed(Plant):
    def __init__(self, world, json=None):
        super().__init__(0, world, Type.MILKWEED, json)

    def construct(self):
        return Milkweed(self._world)

    def action(self):
        super().action()
        super().action()
        super().action()
